CustomMetadata/SamlMetadataCustomConfiguration Example.json

```diff
@@ -0,0 +1,380 @@
+                "Name": "https://data.gov.dk/model/core/eid/professional/orgName",
+                "NameFormat": "urn:oasis:names:tc:SAML:2.0:attrname-format:uri"
+            }
+        ],
+        "UseCustomOrganization": true,
+        "CustomOrganization": {
+            "DisplayNames": [
+                {
+                    "Name": "hhp@globeteam.com",
+                    "Language": "nl"
+                }
+            ],
+            "Names": [
+                {
+                    "Name": "nemlogin3",
+                    "Language": "da"
+                }
+            ],
+            "Urls": [
+                {
+                    "Uri": "http://hhpglobeteam.safewhere.eu",
+                    "Language": "fr"
+                }
+            ]
+        },
+        "UseCustomContactPerson": true,
+        "CustomContactPerson": {
+            "DisplayNames": [
+                {
+                    "Name": "hhp@globeteam.com",
+                    "Language": "nl"
+                }
+            ],
+            "Names": [
+                {
+                    "Name": "nemlogin3",
+                    "Language": "da"
+                }
+            ],
+            "Urls": [
+                {
+                    "Uri": "http://hhpglobeteam.safewhere.eu",
+                    "Language": "fr"
+                }
+            ]
+        },
+        "EncryptionAlgorithms": [
+            "http://www.w3.org/2001/04/xmlenc#aes128-cbc",
+            "http://www.w3.org/2001/04/xmlenc#aes256-cbc",
+            "http://www.w3.org/2009/xmlenc11#aes128-gcm",
+            "http://www.w3.org/2009/xmlenc11#aes192-gcm",
+            "http://www.w3.org/2009/xmlenc11#aes256-gcm"
+        ]
+    },
+    "kombit": {
+        "IncludeArtifactEndpoints": true,
+        "IncludeAuthenticationContextMethodClasses": true,
+        "IncludeContactPerson": true,
+        "NameIdentifierFormats": [
+            "urn:oasis:names:tc:SAML:2.0:nameid-format:transient",
+            "urn:oasis:names:tc:SAML:2.0:nameid-format:persistent"
+        ],
+        "IdentityProviderSupportedAttributes": [
+            {
+                "FriendlyName": "IdentityProviderSupportedAttributes FriendlyName",
+                "Name": "IdentityProviderSupportedAttributes Real Name",
+                "NameFormat": "http://identity-provider-supported-attributes.nameformat.eu",
+                "OriginalIssuer": "IdentityProviderSupportedAttributes OriginalIssuer",
+                "AttributeValueXsiType": "http://www.w3.org/2001/XMLSchema#string",
+                "Values": [
+                    "https://data.gov.dk/model/core/specVersion",
+                    "https://data.gov.dk/concept/core/nsis/loa",
+                    "https://data.gov.dk/model/core/eid/professional/rid",
+                    "https://data.gov.dk/model/core/eid/professional/orgName"
+                ]
+            }
+        ],
+        "ServiceProviderRequestedAttributes": [
```

Comment:
Same issues as the IdentityProviderSupportedAttributes  list

Reply:
Fixed